smalltalksrc/VMMaker/InterpreterPrimitives.class.st

```diff
@@ -904,6 +904,23 @@ InterpreterPrimitives >> primitiveAt [
 	self commonAt: false
 ]
 
+{ #category : #'as yet unclassified' }
+InterpreterPrimitives >> primitiveAtByteArray [
+
+	<accessorDepth: 0>
+	<numberOfArguments: 1>
+	self commonAtWith: [ :array :index | self stByteArray: array at: index ]
+]
+
+{ #category : #'as yet unclassified' }
+InterpreterPrimitives >> primitiveAtPointerArray [
+
+	<accessorDepth: 0>
+	<numberOfArguments: 1>
+	self commonAtWith: [ :array :index |
+		self stPointerArray: array at: index ]
```

Comment:
I like the conciseness :)

We can even refactor the original one to look like this.